retroshare-gui/src/gui/HomePage.cpp

```diff
@@ -50,9 +50,11 @@ HomePage::HomePage(QWidget *parent) :
 	MainPage(parent),
 	ui(new Ui::HomePage)
 {
+    mIncludeAllIPs = false;
```

Comment:
this should be moved in the initializers list, like ui(...)

Reply:
done